src/client.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+export class HelloWorld extends Component {
+  render() {
+    return (
+            <p>Hello World!</p>
```

Comment:
The indentation for a js are quite different from the templates in jsx. Though it just requires that they have consistent opening and closing tags. The reason why this looks off might be because of the component's scanty content. Would check again so that it stays consistent.

Thanks @andela-eisaac